compiler/rustc_passes/src/stability.rs

```diff
@@ -89,6 +91,7 @@ struct Annotator<'a, 'tcx> {
     index: &'a mut Index,
     parent_stab: Option<Stability>,
     parent_const_stab: Option<ConstStability>,
+    parent_body_stab: Option<DefaultBodyStability>,
```

Comment:
This doesn't actually seem to be used for anything (other than being updated as the tree is traversed). I don't think there's ever a case where inheritance would make sense, so I think this can be removed entirely.

Reply:
Removed 👍🏻